Sources/PayPal/Enums/PayPalError.swift

```diff
@@ -0,0 +1,28 @@
+#if canImport(PaymentsCore)
+import PaymentsCore
+#endif
+
+#if canImport(PayPalCheckout)
+@_implementationOnly import PayPalCheckout
```

Comment:
Another slightly side note - we _could_ put these all under the `#if canImport(PaymentsCore)` level if we wanted to make it a little easier to read. 

```
#if canImport(PaymentsCore)
import PaymentsCore
@_implementationOnly import PayPalCheckout
#endif
```